api-docs/src/main/kotlin/org/bitcoindevkit/bdk.kt

```diff
@@ -408,18 +415,18 @@ class TxBuilder() {
      */
     fun enableRbfWithSequence(nsequence: UInt): TxBuilder {}
 
-    /** Finish building the transaction. Returns the BIP174 PSBT. */
-    fun finish(wallet: Wallet): PartiallySignedBitcoinTransaction {}
+    /** Finish building the transaction. Returns a [TxBuilderResult]. */
+    fun finish(wallet: Wallet): TxBuilderResult {}
 }
 
 /**
- * A object holding an address and an amount.
+ * A object holding an ScriptPubKey and an amount.
  *
- * @property address The address.
+ * @property script The ScriptPubKey.
  * @property amount The amount.
  */
 data class AddressAmount (
```

Comment:
Shouldn't this be `ScriptAmount`?
```suggestion
data class ScriptAmount (
```